frontend/vue/components/Syllabus/SyllabusLayout.vue

```diff
@@ -9,23 +9,20 @@
         {{ $translate("This is placeholder text for a page where students are viewing a syllabus.") }}
       </template>
     </UserAccountSectionHeader>
-    <section class="syllabus__content">
```

Comment:
We should also remove the CSS definition of the `syllabus_content` class since it's now no longer used.